### ISSUE TITLE
BUG: Indexing Requiring Manual Sync

### ISSUE THREAD

**Issue by BGLLC** [10up/ElasticPress#3538]
Felipe Elia sent the following reply and closed the ticket as follows:
----------------------------------------------------------------------
Hi @BGLLC,

Looking at the error message, it seems the problem is actually with your server: 

> index [posts] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];

Your disk seems to be too full. The reason why things work after a full sync is that that operation deletes everything (cleaning the disk) and ES unlocks itself. As helping with infrastructure is outside the scope of this repo, I'm going to close the issue now. Thanks!

_Originally posted by @felipeelia in https://github.com/10up/ElasticPress/issues/3527#issuecomment-1620320595_
--------------------------------------------------------------------------------

It would have been more appropriate for him to wait for us to respond prior to closing out the ticket, as we were not given the opportunity to respond with the following:
---------------------------------------------------------------------------------------
"Hello Felipe,

Thank you for your response.  I spoke with our IT Department and it seems the problem was actually created with your most recent update. We operate a large website/database and have been for 13+ years and our clients rely on our site to have information in REAL TIME. I find it hard to believe that we somehow and suddenly reached a data threshold at the exact same time your plugin was updated.

We had ZERO issues with syncing prior to your recent update and we have not made any changes to any of our configurations to our infrastructure."
---------------------------------------------------------------------------------
Now the issue we are having is that even the Fresh Sync is not working properly and not a single one of our customers can find information that has been created or updated more before March 30, 2023.  PLEASE FIX THIS or Revert back to your previous version, because your update has caused nothing but problems.

---------------------------------------------------------------------------------
As an update, the most recent Fresh Sync did not sync more than 2100 posts before spitting out the error "> index [posts] blocked by: [TOO_MANY_REQUESTS/12/disk usage exceeded flood-stage watermark, index has read-only-allow-delete block];"

I have attached the most recent status report.
              
            
[elasticpress-report.txt](https://github.com/10up/ElasticPress/files/12023108/elasticpress-report.txt)

**Comment by brandwaffle** [comment]
Hi @BGLLC ,

As Felipe explained, supporting your custom infrastructure is well beyond the scope of what we provide here.

I appreciate you feel this is a new issue related to an update that may have occurred recently. If this is your theory, I recommend downgrading to a version of the plugin that worked previously and testing to see if this restores your site's search functionality. This would enable you to file a bug report containing the information we request and would enable the community of others who self-host to potentially assist you. To be clear, we are not going to revert the current version of the plugin for a single unverified error report when thousands of others are successfully using this plugin's latest version. That doesn't prevent you from running an earlier version, however, in any way.

As a final note, please keep your comments here within the bounds of good open source etiquette. I understand you may be frustrated, but we are quite clear with our support policies and are simply not able to troubleshoot your infrastructure for you. Any support you're receiving here (including this reply) is provided to help you completely free of charge and without any warranty, just like the open source plugin you're using.

**Comment by felipeelia** [comment]
With all that said, I'm also closing this issue. For those not familiar with GitHub, closed issues still can be commented on, so it is not because an issue was closed that a new issue needs to be created.
